src/app/docs/character-count/character-count-docs.component.html

```diff
@@ -192,4 +238,11 @@
 
   </sky-docs-design-guidelines>
 
+  <sky-docs-code-examples>
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/character-count"
```

Comment:
I think we may need to discuss if we should have a standard pattern here as I've seen various versions of calling this nothing (as you have it), calling it `Basic setup`, and calling this `Character count`. Do you know if we have a standard we should be following?

Reply:
I've been following this pattern - if it has multiple demos, put them in folders. If it has just one demo, then there's no need for folders. How does that sound?

Reply:
Lets discuss in flex time but I'm not too worried about it.